subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -38,26 +32,36 @@ LOG_MODULE_REGISTER(nrf_cloud_rest, CONFIG_NRF_CLOUD_REST_LOG_LEVEL);
 
 #define API_VER				"/v1"
 
-#define CONTENT_TYPE_TXT_PLAIN		"text/plain"
-#define CONTENT_TYPE_ALL		"*/*"
-#define CONTENT_TYPE_APP_JSON		"application/json"
-#define CONTENT_TYPE_APP_OCT_STR	"application/octet-stream"
 #define CONTENT_RANGE_RSP		"content-range: bytes"
```

Comment:
Or was there a reason for this? The server not handling this properly?
```suggestion
#define CONTENT_RANGE_RSP		"Content-Range: bytes"
```

Reply:
only `"content-range` is accepted... i'll ask why.

Reply:
oh, this is the response from the server... it comes back as lower case.

Reply:
Even though this works with a particular server, the HTTP header names are case insensitive, so the comparison should be as well. It would be good practice to send all headers in the same mixed case format as in the RFCs and ignore the case when parsing.